schema/telescope_axis_height.schema.yml

```diff
@@ -0,0 +1,29 @@
+%YAML 1.2
+---
+title: Schema for telescope_axis_height model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/data.metaschema.yml
+meta_schema_version: 0.1.0
+name: telescope_axis_height
+description: Height of telescope elevation axis above ground level.
```

Comment:
Please note that at the moment we do not have this parameter. To get this value from the current values we have we need to subtract (or add) the axes_offset parameter from the CORSIKA sphere height. 

Not to say that this parameter should change, but we need additional information for the actual simulations.

Reply:
Perfect